roles/template_service_broker/tasks/install.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Fact setting
+- name: Set default image variables based on deployment type
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ openshift_deployment_type | default(deployment_type) }}.yml"
+    - "default_images.yml"
+
+- name: set ansible_service_broker facts
+  set_fact:
+    template_service_broker_image: "{{ template_service_broker_image | default(__template_service_broker_image) }}"
+
+- oc_project:
+    name: openshift-template-service-broker
+    state: present
+
+- command: mktemp -d /tmp/tsb-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+  become: no
+
+- name: Pull down files
+  command: >
+    wget {{ __tsb_files_location }}{{ item }} -P {{ mktemp.stdout }}
+  with_items:
+    - "{{ __tsb_template_file }}"
+    - "{{ __tsb_config_file }}"
+    - "{{ __tsb_rbac_file }}"
```

Comment:
Has this stabilized enough that we should go ahead and add this content to openshift-ansible in the location we agreed upon rather than fetching it from origin github?

Reply:
I thought the alternative to pulling the content from origin was going to be a locally sync'd place in ansible. Is that configured yet? We should be able to just change the source location in `vars/`

Reply:
Yeah, we'll setup CI tooling for that but lets go ahead and import a copy of current content as part of this PR?